docker-compose-custom.yml

```diff
@@ -6,11 +6,11 @@ services:
     env_file:
       - .env
     ports:
-      - 30336:30333
-      - 9947:9944
-      - 9936:9933
+      - 30333:30333
```

Comment:
could we increment the port numbers of the other nodes 2 to node 5 like what's done in the proposed brickable PR https://github.com/DataHighway-DHX/node/pull/195/files#diff-b5c7043a49318df5cf0949496bce174d5ad20deb97784bd2571030f8ac38f821R27

Reply:
hm, not sure what you mean, ports are incremented already https://github.com/DataHighway-DHX/node/blob/christian/dockerBuilds/docker-compose-custom.yml

Reply:
yeh i was just thinking 30333, 30334,
instead of                      30333, 30337..
etc

just a nitpick